sympy/assumptions/ask.py

```diff
@@ -38,7 +38,7 @@ def hermitian(self):
         """
         Hermitian predicate.
 
-        ``ask(Q.hermitian(x))`` is true iff ``x`` belongs to the set of
+        ``ask(Q.hermitian(x))`` is true if ``x`` belongs to the set of
```

Comment:
I did search if it has a special meaning but abandoned my search right before the  5th result (the "if and only if") thinking it doesn't. I failed.